drivers/video/video_esp32_dvp.c

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright (c) 2024 espros photonics Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT espressif_esp32_cam
+
+#include <soc/gdma_channel.h>
+#include <soc/lcd_cam_reg.h>
+#include <soc/lcd_cam_struct.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/dma.h>
+#include <zephyr/drivers/dma/dma_esp32.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/video.h>
+#include <zephyr/kernel.h>
+#include <zephyr/pm/device.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(esp32_cam, LOG_LEVEL_INF);
```

Comment:
Currently there is no dedicated log level for video drivers, so setting the log level directly there sounds fine, as it is the role for another pull requests (i.e. that I'll do) to fix this for all video drivers.

Reply:
I would appreciate that to!